### PR TITLE
deployment template diff func add

### PR DESCRIPTION
**Pull request by BennieMeng**
**What type of PR is this?**
> Uncomment only one, leave it on its own line:
>
> /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes https://github.com/Qihoo360/wayne/issues/137

**Special notes for your reviewer**:
image

```release-note
add deployment template diff
line or side translate
yaml or json translate
```

### CODE REVIEW

**Comment by wilhelmguo** [comment]
@BennieMeng 增加一下Modal 统一的放大按钮，
image

**Comment by BennieMeng** [comment]
done

File: src/frontend/src/app/portal/configmap/list-configmap/list-configmap.component.ts [hunk 1]
```diff
@@ -74,6 +77,10 @@ export class ListConfigMapComponent implements OnInit, OnDestroy {
   ngOnInit(): void {
   }
 
+  diffTmp() {
```
Comment (chengyumeng) [inline src/frontend/src/app/portal/configmap/list-configmap/list-configmap.component.ts]: Tmp 是temp，临时的意思，这个是diffTemplate

Comment (wilhelmguo) [reply src/frontend/src/app/portal/configmap/list-configmap/list-configmap.component.ts]: Wayne里Template都缩写为Tpl

Comment (BennieMeng) [reply src/frontend/src/app/portal/configmap/list-configmap/list-configmap.component.ts]: 这么别扭的吗

**Pull request merged**